test/service/test_response.py

```diff
@@ -1289,6 +1295,8 @@ def test_file_response(self):
         }
         file = one(one(keyword_response['hits'])['files'])
         self.assertElasticsearchResultsEqual(file, expected_file)
+        project = one(one(keyword_response['hits'])['projects'])
+        self.assertEqual(project['documentId'], ['88ec040b-8705-4f77-8f41-f81e57632f7d'])
```

Comment:
Instead of adding the test here where only the `files` response is being tested, and in `test_sample` where only the `samples` endpoint is being tested, how about adding the assert in `test_filter_by_projectId` and cover all endpoints and be able to assert by value.

```
diff --git a/test/service/test_response.py b/test/service/test_response.py
index fa8dfebe..d7deaed0 100644
--- a/test/service/test_response.py
+++ b/test/service/test_response.py
@@ -1347,8 +1347,10 @@ class TestResponse(WebServiceTestCase):
                         for project in hit['projects']:
                             if entity_type == 'projects':
                                 self.assertEqual(test_data['title'], project['projectTitle'])
+                                self.assertEqual(test_data['id'], project['documentId'])
                             else:
                                 self.assertIn(test_data['title'], project['projectTitle'])
+                                self.assertIn(test_data['id'], project['documentId'])
                     for term in response_json['termFacets']['project']['terms']:
                         self.assertEqual(term['projectId'], [test_data['id']])
```